app/src/main/java/com/example/workoutplaner/MainActivity.java

```diff
@@ -2,13 +2,35 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
 
 public class MainActivity extends AppCompatActivity {
 
+    //private Button open_nav;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+
+        // paprastas button, kuris yra main activity
+        /*open_nav = (Button) findViewById(R.id.button);    *mygtukas, kol kas istrintas, tai neveiks be button id
+        open_nav.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                openNavigation();       // atidaro navigation
+
+            }});*/
```

Comment:
I do not think that changes made to main activity, just for testing purposes, should be included in a commit.